### PR TITLE
Filled out the headings and basic button styling

### PR DESCRIPTION
**Pull request by alafritz**
I still need to verify and tweak them, which we will do tomorrow.

### CODE REVIEW

File: src/less/components/headings.less [hunk 1]
```diff
@@ -1,19 +1,67 @@
+p {
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  margin-top: 1rem;
+  margin-bottom: 3rem;
+}
+
 h1 {
-    font-size: 4rem;
```
Comment (antsav) [inline src/less/components/headings.less]: u can use less mixins for repetitive parts of styles http://lesscss.org/features/#mixins-feature

File: src/less/components/headings.less [hunk 2]
```diff
@@ -1,19 +1,67 @@
...
+.quote {
+  font-size: 1rem;
+  font-weight: 400;
+  font-style: italic;
+  text-transform: uppercase;
+  line-height: 1.5rem;
+  margin-left: 2rem;
+  margin-top: 1rem;
+  margin-right: 2rem;
+  margin-bottom: 1rem;
```
Comment (antsav) [inline src/less/components/headings.less]: margin: 1rem 2rem;

File: src/less/core/variables.less [hunk 3]
```diff
@@ -3,7 +3,7 @@
 @cRed:      #D50000;
 @cGreen: #2ECC51;
 @cGrey:     #676767;
-@cBackgroundGrey:     #CFCFCF;
+@cBackgroundGrey:     #EFEFEF;
```
Comment (antsav) [inline src/less/core/variables.less]: nice 👍

**Comment by alafritz** [comment]
What are your thoughts on pulling out the scaling of the elements. So instead of adding max & min width to each component, we have some separate component document that only handles the scaling.
I think we breakdown the components to very simple elements so people combine them accordingly. How do you feel?

**Comment by alafritz** [comment]
Hey, i figured out mixins and i also sorted some other css stuff out.

**Pull request closed without merging**